pkg/osbuild/mkswap_stage.go

```diff
@@ -0,0 +1,16 @@
+package osbuild
+
+type MkswapStageOptions struct {
+	UUID  string `json:"uuid"`
+	Label string `json:"label,omitempty"`
+}
+
+func (MkswapStageOptions) isStageOptions() {}
+
+func NewMkswapStage(options *MkswapStageOptions, devices map[string]Device) *Stage {
```

Comment:
(niptick/followup) - I know its a may sound a bit silly but a tiny test like:
```go
package osbuild

import (
	"encoding/json"
	"testing"

	"github.com/stretchr/testify/assert"
)

var expectedJSON = `{
  "type": "org.osbuild.mkswap",
  "options": {
    "uuid": "8a1fc521-02a0-4917-92a9-90a44d7e6503",
    "label": "some-label"
  },
  "devices": {
    "root": {
      "type": "org.osbuild.loopback"
    }
  }
}`

func TestNewMkswapStage(t *testing.T) {
	devices := make(map[string]Device)
	devices["root"] = Device{
		Type: "org.osbuild.loopback",
	}

	options := MkswapStageOptions{
		UUID:  "8a1fc521-02a0-4917-92a9-90a44d7e6503",
		Label: "some-label",
	}
	stage := NewMkswapStage(&options, devices)
	b, err := json.MarshalIndent(stage, "", "  ")
	assert.NoError(t, err)
	assert.Equal(t, expectedJSON, string(b))
}

```  
(also in the tests-as-documentation sense)

Reply:
Done.